scripts/curate_basel_additional_data.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import json
+import argparse
+
+
```

Comment:
Personally, I could use a docstring here mentioning the upstream source of the data this script is meant to be run on and/or it's intermediate location on `duke` (if it has one).